lib/mapUtils.js

```diff
@@ -155,7 +152,8 @@ function computeNextOpenDay(todaysDayIndex, hoursPerDayDict) {
   return 'Closed until '
     .concat(tmrwDay)
     .concat(' ')
-    .concat(tmrwOpeningTime);
+    .concat(tmrwOpeningTime)
+    .replace('12am', 'Midnight');
```

Comment:
I'm p sure you actually wouldn't need this here or in 'Open now until' below bc doing this in the final `StoreHours` does the final change.

Reply:
Oh I added it there because I don't display `StoreHours` on the `StoreCard` (this method primarily gives you the summary, aka "Open 24/7" or whatever, and `StoreHours` gives you the whole schedule)